app/assets/javascripts/capture.js

```diff
@@ -1,5 +1,15 @@
 //= require graph.js
-// window.webcam.getCameraList()
```

Comment:
OK, so here, we shouldn't need to change this file at all - in fact, we should be able to point the new html file at the newly upgraded library from https://github.com/publiclab/spectral-workbench/pull/668 -- and perhaps for the time being we should avoid any changes to `application.js` so as to not touch any existing working systems. By separating out the included JS and CSS files for the new interface, we keep the dependencies for each interface separate until we're sure it's all working. How does that sound?

Reply:
I thought maybe the old capture interface could benefit from the new integration since that would also allow for cross browser support on the old interface along with the new one...